### DISCUSSION TITLE
SystemFonts.Families.Any() returns false on Linux

### DISCUSSION THREAD

**Discussion by nikkilocke** [SixLabors/Fonts#414] [Q&A]
If I run my C# project which uses SixLabors.Fonts on Windows, there are plenty of fonts available to me.

On Linux (debian), there are none.

The code is running in a docker container mcr.microsoft.com/dotnet/runtime:7.0.

Why are there no fonts, and how do I go about adding one?

**Comment by tocsoft** [comment]
are you sure the `mcr.microsoft.com/dotnet/runtime:7.0` image has fonts installed in it? I would personally not expect a stripped down base docker image to ship with any fonts at all.

**Comment by tocsoft** [comment]
I would recommend shipping any font you are dependent on as part of your application and using a manually configure `FontCollection` (a global static/singleton would be my recommendation for handling this configured collection).
